server/dfr3-service/src/main/java/edu/illinois/ncsa/incore/service/dfr3/controllers/FragilityController.java

```diff
@@ -267,8 +279,8 @@ public FragilitySet uploadFragilitySet(@ApiParam(value = "json representing the
     @DELETE
     @Produces(MediaType.APPLICATION_JSON)
     @Path("{fragilityId}")
-    @ApiOperation(value = "Deletes a fragility by id")
-    public FragilitySet deleteFragilityById(@ApiParam(value = "fragility id", example = "5b47b2d8337d4a36187c6727") @PathParam(
+    @Operation(summary = "Deletes a fragility by id")
```

Comment:
Maybe take a look at the [changes for space service](https://github.com/IN-CORE/incore-services/pull/153/files#diff-c9a04636dc9c781379370b2e9d250052407911e814fdf57c4f3c6d9c5ad24ef9)? I found @ApiOperation is different from the space